### ISSUE TITLE
Annotations can be moved

### ISSUE THREAD

**Issue by RahelaPogacean** [annotorious/annotorious-openseadragon#183]
Hello! I am developing an application using Annotorious with OSD and I can move the annotations with drag & drop. Is there any possibility to disable this option?

**Comment by rsimon** [comment]
The `readOnly` option is probably what you're looking for:
https://annotorious.github.io/api-docs/osd-plugin/#readonly

**Comment by RahelaPogacean** [comment]
@rsimon  Yes, but readOnly option does not allow me to edit the annotation anymore.

**Comment by rsimon** [comment]
If you want to make the shape locked, but keep the comments editable, I think you can do something like this:

```js
var anno = Annotorious.init({
  image: 'hallstatt',
  readOnly: true,
  widgets: [
    { widget: 'COMMENT', editable: true }
  ]
});
```

But actually not quite sure anymore... Annotorious v3 is already on the horizon and will allow for more flexibility concerning permissions and editing rules.

**Comment by RahelaPogacean** [comment]
@rsimon I want to not be able to move the annotation, but still be able to edit the points

**Comment by rsimon** [comment]
Ok, understood. Sorry - no, moving the shape but not editing it is not possible. You might have a chance by using CSS to hide the resize handles perhaps.

**Comment by ferropasha** [comment]
Is the reverse possible? That is for a user to be able to move selection frame around, but not to change it's shape?
